source/clear-linux/get-started/compatibility-check.rst

```diff
@@ -1,6 +1,6 @@
 .. _compatibility-check:
 
-Check system compatibility with Clear Linux OS for Intel® Architecture
+Check processor and EFI firmware compatibility with Clear Linux OS for Intel® Architecture
```

Comment:
This title is now too long. I will change |CLOSIA| to |CL| to avoid having title issues in the sidebars, etc. Also, the == below making this a L3 header should match in length with the title.